oeps/oep-0052/decisions/002-schema-specification.rst

```diff
@@ -0,0 +1,30 @@
+Event Bus Messages will be Defined in AVRO Specification Language
```

Comment:
Note: This ADR was also rolled into the other ADR. See https://github.com/openedx/openedx-events/blob/main/docs/decisions/0005-external-event-schema-format.rst